### ISSUE TITLE
Ctrl-z doesn't output correct Scenetrans setup code

### ISSUE THREAD

**Issue by sebjameswml** [ABRG-Models/morphologica#203]
Especially 0 values come out as 0f rather than 0.0f.

Solution might be to output float{val}

**Comment by sebjameswml** [comment]
Example:

```
Scenetrans setup code:
    v.setSceneTrans (morph::vec<float,3>({0f, 0f, -3.5f}));
scene rotation is Quaternion[wxyz]=(1,0,0,0)
Writing scene trans/rotation into /tmp/Visual.json... Success.

```

**Comment by sebjameswml** [comment]
Fixed with 64b7cc8
